snapshots/who/2024-07-26/mortality_database.csv.dvc

```diff
@@ -0,0 +1,35 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: WHO Mortality Database
+    description: |-
+      The WHO mortality database is a collection death registration data including cause-of-death information from member states.
+
+          Where they are collected, death registration data are the best source of information on key health indicators, such as life expectancy, and death registration data with cause-of-death information are the best source of information on mortality by cause, such as maternal mortality and suicide mortality.
+
+          WHO requests from all countries annual data by age, sex, and complete ICD code (e.g., 4-digit code if the 10th revision of ICD was used). Countries have reported deaths by cause of death, year, sex, and age for inclusion in the WHO Mortality Database since 1950.
+
+          Data are included only for countries reporting data properly coded according to the International Classification of Diseases (ICD). Today the database is maintained by the WHO Division of Data, Analytics and Delivery for Impact (DDI) and contains data from over 120 countries and areas. Data reported by member states and selected areas are displayed in this portal’s interactive visualizations if the data are reported to the WHO mortality database in the requested format and at least 65% of deaths were recorded in each country and year.
+    date_published: "2024-02-21"
+
+    # Citation
+    producer: WHO Mortality Database
+    citation_full: |-
+      WHO Mortality Database, Data, Analytics & Delivery for impact (DDI), World Health Organization (2024)
```

Comment:
tiniest nitpick, but they refer to themselves as "WHO Division of Data, Analytics and Delivery for Impact (DDI)", so maybe I would stick to that (except if the citation is given somewhere like that, but I couldn't find it)

Reply:
Yeh that's a good point, I'll use "WHO Division of Data, Analytics and Delivery for Impact (DDI)"  instead